src/field_theory/minimal_polynomial.lean

```diff
@@ -64,6 +64,13 @@ le_of_not_lt $ well_founded.not_lt_min degree_lt_wf _ hx ⟨pmonic, hp⟩
 lemma ne_zero [nontrivial α] : (minimal_polynomial hx) ≠ 0 :=
 ne_zero_of_monic (monic hx)
 
+/-- If an element `x` is a root of a nonzero monic polynomial `p`,
+then the degree of `p` is at least the degree of the minimal polynomial of `x`. -/
+lemma degree_le_of_monic
+  {p : polynomial α} (hmonic : p.monic) (hp : polynomial.aeval x p = 0) :
+  degree (minimal_polynomial hx) ≤ degree p :=
+min _ hmonic (by simp [hp])
+
```

Comment:
Could you explain (in the docstring) what the difference is between `degree_le_of_monic` and `min`? It looks like `hp` are exactly the same (not just equal up to `simp`) in both cases.

Reply:
Ah, you're right, the two theorem are exactly the same. I will remove `degree_le_of_monic`. I don't know why I added `degree_le_of_monic` 🤔